1_fetch/src/get_state_inventory.R

```diff
@@ -0,0 +1,3 @@
+get_state_inventory <- function(sites_info, state) {
```

Comment:
You might consider renaming to `filter_state_inventory()`. Using "get" makes it sound like it is the step that queries the web service (especially when it is next to the function `get_site_data()`), but really it is just filtering existing data to keep only a single state's inventory info.

Reply:
Another option is to leave top-level in the makefile since it is only one line of code. E.g.

```r
tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb))
```

Reply:
I like leaving it top-level in the makefile, it felt silly making that a function 👍🏽